### PR TITLE
Fix 182: changed reverb attribute codes, labels and a little housekee…

### PR DESCRIPTION
**Pull request by zztimur**
Fix for issue #182.

- Attribute id prefixes changed from rev_ to reverb_
- Attribute labels are updated from Rev* to Reverb*
- Some basic clean up of setup scripts
- Code adjustments due to rev->reverb modifications

### CODE REVIEW

**Comment by skwp** [comment]
Awesome thanks for the contribution!

File: app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Reverb
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php]: Can we please exclude licensing info from the files? The entire project will be licensed under Apache 2.0

Comment (zztimur) [inline app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php]: Removed license info

File: app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php [hunk 2]
```diff
@@ -55,22 +55,22 @@ public function getTargetProductId() {
         try {
             $productId = Mage::registry('current_product') -> getId();
             $targetProductId = Mage::getModel('catalog/product') -> load($productId);
-            $revProductId = $targetProductId -> getRevProductId();
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php]: this variable didn't get renamed

File: app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php [hunk 3]
```diff
@@ -55,22 +55,22 @@ public function getTargetProductId() {
...
         }
-        return $revProductId;
+        return $reverbProductId;
     }
     public function getTargetProductUrl() {
         try {
             $productId = Mage::registry('current_product') -> getId();
             $targetProductId = Mage::getModel('catalog/product') -> load($productId);
-            $revProductUrl = $targetProductId -> getRevProductUrl();
+            $revProductUrl = $targetProductId -> getReverbProductUrl();
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Block/Adminhtml/Catalog/Product/Tab.php]: same rename problem here I think

**Comment by skwp** [comment]
Thanks for the contrib, it seems we have some merge conflicts now though. Also how did you test this? It seems to me there are two possible bugs there with the variable not renamed inside getTargetProductId and getTargetProductUrl? Thanks

**Comment by zztimur** [comment]
Hey @skwp,
I fixed the stuff you pointed out. Thanks.

I tested after I deleted the module, completely dropped reverb tables, removed all resource references.

What's the conflict?

**Comment by skwp** [comment]
not sure github is showing a merge conflict. let me check it out

**Comment by skwp** [comment]
Merged my own branch of thsi thanks

**Pull request closed without merging**